otelconfig/otelconfig.go

```diff
@@ -411,11 +420,17 @@ func newResource(c *Config) *resource.Resource {
 
 	attributes = append(r.Attributes(), attributes...)
 
+	baseOptions := []resource.Option{
+		resource.WithSchemaURL(semconv.SchemaURL),
+		resource.WithAttributes(attributes...),
+	}
+
+	options := append(baseOptions, c.ResourceOptions...)
+
 	// These detectors can't actually fail, ignoring the error.
 	r, _ = resource.New(
```

Comment:
It seems like this comment is no longer true, as there are arbitrary Options here now, right? Does this need to do something with the error?